contracts/multiply/MultiplyProxyActions.sol

```diff
@@ -785,35 +746,36 @@ contract MultiplyProxyActions {
     );
   }
 
-  function executeOperation(
-    address[] calldata assets,
-    uint256[] calldata amounts,
-    uint256[] calldata premiums,
+  function onFlashLoan(
     address initiator,
```

Comment:
this value is supposed to be set to what ? shouldn't we check it to validate right call source?